homeassistant/helpers/condition.py

```diff
@@ -134,23 +141,20 @@ def if_and_condition(
 and_from_config = _threaded_factory(async_and_from_config)
```

Comment:
This is now broken because `async_and_from_config` is a coroutine function. Is this used anywhere?

Reply:
Removed.